db/seeds.d/07-provisioning_templates.rb

```diff
@@ -79,6 +80,13 @@
       :snippet  => false,
       :template => File.read(File.join("#{Rails.root}/app/views/unattended", input.delete(:source)))
     }.merge(input))
+
+    created.push(t.name)
```

Comment:
Not sure what you mean by "template taxonomies". The reason I'm doing it this way is to support reoccurring seed runs, if user disassociate the template we should not re-associate it so we only associate those that were newly created in this seed run